content/projects/nodejs/sql/_index.md

```diff
@@ -59,4 +59,20 @@ You will be expected to properly test your code. You can use whatever testing fr
 ## Resources
 
 - https://www.guru99.com/introduction-to-database-sql.html
-- https://www.w3schools.com/sql/sql_intro.asp
\ No newline at end of file
+- https://www.w3schools.com/sql/sql_intro.asp
+
+## Instructions for reviewers
+
+- Connection strings should never be exposed. Ensure that the `.dotenv` module has been utilized.
+
+- The `.dotenv` file should exist in the `.gitignore` file.
```

Comment:
Could we maybe reword this line. `The .dotenv file should not be present in the repo and should be specifically excluded through use of .gitignore.` I think that make more sense but correct me if I am wrong?